From 083633fb21c4579b6326923b74d1a90b17d9635e Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Tue, 7 Nov 2017 11:26:56 +0200
Subject: [PATCH 2730/5242] MLK-16742-3: dts: arm64: imx8qm: Fix ESAI fsys
 clock.

commit  288c9014dd2d0cd2f09bf539ec65b575615521db from
https://source.codeaurora.org/external/imx/linux-imx.git

Set ESAI fsys clock to IMX8QM_AUD_ESAI_0_IPG.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 2189029..01f1e04 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -2586,7 +2586,7 @@
 		interrupts = <GIC_SPI 409 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8QM_AUD_ESAI_0_IPG>,
 			<&clk IMX8QM_AUD_ESAI_0_EXTAL_IPG>,
-			<&clk IMX8QM_CLK_DUMMY>,
+			<&clk IMX8QM_AUD_ESAI_0_IPG>,
 			<&clk IMX8QM_CLK_DUMMY>;
 		clock-names = "core", "extal", "fsys", "spba";
 		dmas = <&edma2 6 0 1>, <&edma2 7 0 0>;
-- 
1.7.9.5

